pkg/utils/kubernetes/kubernetes.go

```diff
@@ -79,6 +79,20 @@ func CreateOrUpdate(ctx context.Context, c client.Client, obj client.Object, f c
 	return controllerutil.CreateOrUpdate(ctx, c, obj, f)
 }
 
+// Find and return the secrets referenced by secretRefs
```

Comment:
```suggestion
// ResolveSecrets finds and returns the secrets referenced by secretRefs
```
see the go doc best practices: https://blog.golang.org/godoc

Reply:
Done (by amending to the original commit).
How relevant is 
> Amend the affected commit(s) and force push onto your branch.
from the [Gardener Pull Request checklis](https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md#Pull-Request-Checklist)?

Reply:
i will anyway squash the commits, so thats fine